pkg/catalog-cd/fetcher/config/testdata/contract.multiple.yaml

```diff
@@ -0,0 +1,12 @@
+tasks:
```

Comment:
This is what a contracts looks like. This is what would be "required" to be attached on external repostiory releases (`bundle` is optionnal for example).

Reply:
The contract could live as a dot file on the root of the repository, thus when the "fetcher" loads the repository it picks the meta-information required.

Reply:
So right now, the contract lives in each releases, mainly as some things might change from releases to releases (the number of task/pipeline might differ, or the way to run tests, …).

That said, it could also be split into several contracts/files.